css/style.css

```diff
@@ -0,0 +1,295 @@
+  color: darkgray;
+}
+
+.second-section {
+  width: 100%;
+  height: auto;
+  background-color: #272a31;
+}
+
+.card {
+  background-color: black;
+  width: 90%;
+  height: auto;
+  margin-top: 1rem;
+  border-radius: 0;
+  margin-left: 0.2rem;
+  margin-right: 0.2rem;
+  text-align: center;
+}
+
+.icons {
+  font-size: 3rem;
+  padding-top: 1rem;
+  color: #fff;
+}
+
+.img-border {
+  width: 5rem;
+  height: 5rem;
+}
+
+.aseel-images {
+  position: relative;
+  max-width: 10rem;
+}
+
+.aseel-img-container {
+  position: absolute;
+  top: 1rem;
+  left: 1rem;
+  width: 9rem;
+}
+
+.aseel-img {
+  width: 100%;
+  height: 10rem;
+}
+
+.img-content {
+  width: 100%;
+  height: auto;
+  margin-left: 7rem;
+}
+
+.fourth-section {
+  background-color: #272a31;
+  width: 100%;
+  height: 400px;
+}
+
+.media-img {
+  width: 80%;
+  height: 60%;
+  filter: brightness(50%);
+}
+
+footer {
+  width: 100%;
+  height: 200px;
+}
+
+.footer-img {
+  width: 50%;
+  height: 50%;
+}
+
+@media screen and (min-width: 786px) {
+  #navbarNav {
+    margin-left: 26rem;
+  }
+
+  .nav-item {
+    padding-right: 3rem;
+  }
+
+  .first-navbar-list {
+    display: flex;
+    justify-content: end;
+  }
+
+  #black-navbar {
+    background-color: #272a31;
+  }
+
+  .navbar-brand {
+    color: #fff;
+  }
+
+  .active {
+    color: #ec5242;
+    text-decoration: none;
+    display: block;
+    padding: var(--bs-nav-link-padding-y) var(--bs-nav-link-padding-x);
+  }
+
+  .ReadMoreBtn {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .mobile-row {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .navbar-toggler {
+    position: absolute;
+    top: 1rem;
+  }
+
+  #logo {
+    max-width: 100%;
+    height: 40px;
+    filter: brightness(50%);
+    -webkit-filter: brightness(50%);
+    margin-left: 21rem;
+  }
+
+  .aseel-name {
+    color: #272a31;
+    border: none;
+  }
+
+  #black-navbar {
+    order: 1;
+
+    --bs-bg-opacity: 1;
+
+    background-color: rgba(var(--bs-tertiary-bg-rgb), var(--bs-bg-opacity));
+  }
+
+  .navbar-brand {
+    color: #272a31;
+  }
+
+  .active {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .navbar-toggler:focus {
+    outline: none;
+    box-shadow: none;
+  }
+
+  .navbar-collapse {
+    padding-left: 1rem;
+  }
+
+  .card-p {
+    margin-left: 3rem;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    text-align: left;
+    width: 80%;
+  }
+
+  .icons {
+    padding-left: 1rem;
+  }
+
+  .card-title {
+    margin-left: 22px;
+  }
+
+  .ReadMoreBtn {
+    display: block;
+    text-align: center;
+    width: 100%;
+    margin-bottom: 1rem;
+    border: 1px solid #d3d3d3;
+  }
+
+  .card-btn {
+    background-color: #ec5242;
+    text-decoration: none;
+    padding: 2rem 4rem;
+    font-weight: bold;
+  }
+}
+
+.about-p {
+  width: 100%;
+  height: auto;
+  border: 2px solid #fff;
+  background-color: #fff;
+}
+
+.second-section-about {
+  border: 1px solid #d3d3d3;
+}
+
+.about-aseel-logo {
+  width: 40%;
```

Comment:
- [ ] It will be better to remove the width from the image and let it take the default width because in the mobile version the image is compressed